client/src/components/brushableHistogram/index.js

```diff
@@ -4,439 +4,20 @@ https://bl.ocks.org/mbostock/34f08d5e11952a80609169b7917d4172
 https://bl.ocks.org/SpaceActuary/2f004899ea1b2bd78d6f1dbb2febf771
```

Comment:
What is the significance of these comments?  They seem like notes from the component development phase?